paddle/operators/mul_op.h

```diff
@@ -14,17 +14,30 @@
 
 #pragma once
 
-#include <glog/logging.h>
-#include <paddle/framework/operator.h>
+#include "glog/logging.h"
+#include "paddle/framework/eigen.h"
+#include "paddle/framework/operator.h"
 
 namespace paddle {
 namespace operators {
 
-template <typename Place>
+template <typename Place, typename T>
```

Comment:
I am thinking on it too, and haven't get a conclusion. And I will have a discussion with colleagues.